jenkins-client/src/main/java/com/offbytwo/jenkins/client/JenkinsHttpClient.java

```diff
@@ -161,7 +162,11 @@ public JenkinsHttpClient(URI uri, String username, String password) {
      */
     public <T extends BaseModel> T get(String path, Class<T> cls) throws IOException {
         HttpGet getMethod = new HttpGet(api(path));
+        
+        System.out.println(getMethod.getURI());
+        System.out.println(getMethod.getRequestLine());
```

Comment:
System.out is never good idea in production code...better use logging..? I have introduced a logging in https://github.com/jenkinsci/java-client-api/commit/cf652797a7fec49fcaa9cbc1850ad461f951db14#diff-f14ef3181c6e40c08aac9c02ba835778....